src/css/custom.css

```diff
@@ -87,4 +87,16 @@ a:hover {
 
 .footer{
   background-color: #001;
-}
\ No newline at end of file
+}
+
+.bg-white {
+  width: 100%; /* Ensure the background div is full width */
+}
+
+.table-auto {
+  width: 100%; /* Ensure the table is full width */
+}
+
+.container {
+  max-width: none; /* Ensure the container does not restrict width */
+}
```

Comment:
I suggest deleting the CSS code you added. We are using Tailwind CSS, so you can apply its class names instead.

Reply:
Okay...I can make changes with class names without writing css.

I will fix it soon.